tests/unittests/library/test_nsx_security_policy.py

```diff
@@ -0,0 +1,317 @@
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import unittest
+import mock
+import pytest
+import cloudify_nsx.library.nsx_security_policy as nsx_security_policy
+from cloudify import exceptions as cfy_exc
+
+
+class NsxSecurityPolicyTest(unittest.TestCase):
+
+    @pytest.mark.internal
+    @pytest.mark.unit
+    def test_add_policy_group_bind(self):
+        """Check nsx_security_policy.add_policy_group_bind func"""
+        def prepere_check():
```

Comment:
There's a fair amount of shared code with the previous file here- maybe this should all be factored into some common test methods?